tooling/reproducible/linux_repro_build_compare.sh

```diff
@@ -126,23 +233,22 @@ BOOTJDK_VERSION=$(jq -r '.metadata.tools[] | select(.name == "BOOTJDK") | .versi
 GCCVERSION=$(jq -r '.metadata.tools[] | select(.name == "GCC") | .version' "$SBOM" | sed 's/.0$//')
 LOCALGCCDIR=/usr/local/gcc$(echo "$GCCVERSION" | cut -d. -f1)
 TEMURIN_BUILD_SHA=$(jq -r '.components[0] | .properties[] | select (.name == "Temurin Build Ref") | .value' "$SBOM" | awk -F/ '{print $NF}')
-TEMURIN_BUILD_ARGS=$(jq -r '.components[0] | .properties[] | select (.name == "makejdk_any_platform_args") | .value' "$SBOM" | cut -d\" -f4 | sed -e "s/--disable-warnings-as-errors --enable-dtrace --without-version-pre --without-version-opt/'--disable-warnings-as-errors --enable-dtrace --without-version-pre --without-version-opt'/" -e "s/ --disable-warnings-as-errors --enable-dtrace/ '--disable-warnings-as-errors --enable-dtrace'/" -e 's/\\n//g' -e "s,--jdk-boot-dir [^ ]*,--jdk-boot-dir /usr/lib/jvm/jdk-$BOOTJDK_VERSION,g")
+TEMURIN_BUILD_ARGS=$(jq -r '.components[0] | .properties[] | select (.name == "makejdk_any_platform_args") | .value' "$SBOM")
 TEMURIN_VERSION=$(jq -r '.metadata.component.version' "$SBOM" | sed 's/-beta//' | cut -f1 -d"-")
-
+BUILDSTAMP=$(jq -r '.components[0].properties[] | select(.name == "Build Timestamp") | .value' "$SBOM")
 NATIVE_API_ARCH=$(uname -m)
 if [ "${NATIVE_API_ARCH}" = "x86_64" ]; then NATIVE_API_ARCH=x64; fi
 if [ "${NATIVE_API_ARCH}" = "armv7l" ]; then NATIVE_API_ARCH=arm; fi
 
 checkAllVariablesSet
-
 downloadTooling
 setEnvironment
```

Comment:
split this into two:
```
setAntEnvironment()

if [[ NOT_USING_DEVKIT ]]; then
  setGccEnvironment()
fi
```

Reply:
in downloadTooling() add DEVKIT check... :
```
if [[ NOT_USING_DEVKIT ]]; then
    if [ ! -r "${LOCALGCCDIR}/bin/g++-${GCCVERSION}" ]; then
    echo "Retrieving gcc $GCCVERSION" && curl "https://ci.adoptium.net/userContent/gcc/gcc$(echo "$GCCVERSION" | tr -d .).$(uname -m).tar.xz" | (cd /usr/local && tar xJpf -) || exit 1
  fi
fi
```

Reply:
@andrew-m-leonard how is `NOT_USING_DEVKIT` defined? Is it defined by following options?

```

if [[ "${CONFIGURE_ARGS}" =~ .*"--with-devkit=".* ]]; then
  echo "Using gcc from DevKit toolchain specified in configure args"
elif [[ "${BUILD_ARGS}" =~ .*"--use-adoptium-devkit".* ]]; then
  echo "Using gcc from Adoptium DevKit toolchain specified in --use-adoptium-devkit build args"
```

Reply:
SBOM has the args `--use-adoptium-devkit`

Reply:
> @andrew-m-leonard how is `NOT_USING_DEVKIT` defined? Is it defined by following options?
> 
> ```
> 
> if [[ "${CONFIGURE_ARGS}" =~ .*"--with-devkit=".* ]]; then
>   echo "Using gcc from DevKit toolchain specified in configure args"
> elif [[ "${BUILD_ARGS}" =~ .*"--use-adoptium-devkit".* ]]; then
>   echo "Using gcc from Adoptium DevKit toolchain specified in --use-adoptium-devkit build args"
> ```

Correct, --use-adoptium-devkit will download the required devkit